### ISSUE TITLE
Training loss (avg_loss) on Custom data set is very high and keeps increasing

### ISSUE THREAD

**Issue by Karthik-Suresh93** [AlexeyAB/darknet#678]
Hi,

I am training YOLOv3 for my custom data set. The error seems to be too high initially (900 ish) and it increases to values above 1000 in about 30 iterations. This happened in both the original darknet and this fork. I have 963 images and each of them has about 2-3 objects. I have only one class (car). Here is the snapshot of training
![selection_004](https://user-images.githubusercontent.com/25709940/39082468-27091a0e-4519-11e8-838e-19811904400a.png)

Here is the snapshot of my cfg file

![selection_006](https://user-images.githubusercontent.com/25709940/39082482-63aea780-4519-11e8-9207-bd400a4b65e7.png)

Here is the snapshot of my makefile

![selection_008](https://user-images.githubusercontent.com/25709940/39082502-ac65b496-4519-11e8-8e17-ea875a3af1d0.png)

I have set random=1, I have not changed the architecture of the model. I also calculated custom anchors using calc_anchors for 448x448 image. I do not know what is going wrong. Please help me.

Thanks and Regards,
Karthik

**Comment by Karthik-Suresh93** [comment]
After 2400 iterations the loss is now constant around 0.14. Is this good enough to stop? Also, what does the parameter burn_in mean?

**Comment by AlexeyAB** [comment]
@Karthik-Suresh93 Train about 6000 - 8000 iterations and check mAP.
